lib/stones-spec/example.rb

```diff
@@ -37,11 +41,15 @@ def stop!
       [@actual_final_board_file, @initial_board_file].each { |it| it.unlink }
     end
 
+    def default_title
+      @subject.default_title language, source, @precondition.arguments
+    end
+
     private
 
     def make_error_output(error_message, initial_board_gbb)
       if language.is_runtime_error?(@result)
-        "#{get_html_board 'Tablero inicial', initial_board_gbb}\n#{error_message}"
```

Comment:
I agree it is a bit difficult to follow, but I didn't like the "manually copy approach" that was before.